whats_new/ose_3_0_release_notes.adoc

```diff
@@ -93,3 +93,149 @@ link:../rest_api/kubernetes_v1.html#v1-rbdvolumesource[RADOS (Ceph)].
+
+xPaaS images for OpenShift Enterprise 3.0 are provided for Red Hat JBoss
+Enterprise Application Platform, Red Hat JBoss Web Server and Red Hat
+JBoss A-MQ.
+
+== xPaaS Images
+
+=== xPaaS Image for Red Hat JBoss EAP
+
+
+With this release, Red Hat JBoss EAP is available as a containerized
+xPaaS image that is designed for use with OpenShift Enterprise 3.0.
+
+However, there are significant differences in supported configurations
+and functionality in the JBoss EAP xPaaS image compared to the regular
+release of JBoss EAP. To learn about these differences, please read
+https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-using-images#jboss-eap[this documentation].
+Documentation for other JBoss EAP functionality not specific to the
+JBoss EAP xPaaS image can be found in the https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/[JBoss
+EAP documentation on the Red Hat Customer Portal].
+
+=== xPaaS Image for Red Hat JWS
+
+With this release, the Apache Tomcat 7 and Apache Tomcat 8 components of
+Red Hat JBoss Web Server 3 are available as containerized xPaaS images
+that are designed for use with OpenShift Enterprise 3.0.
+
+However, there are significant differences in the functionality between the JBoss Web
+Server xPaaS images and the regular release of JBoss Web Server. To
+learn about these differences, please read
+https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-using-images#jboss-web-server[this documentation].
+Documentation for other JBoss Web Server functionality not specific to
+the JBoss Web Server xPaaS images can be found in the
+https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Web_Server/[JBoss
+Web Server documentation on the Red Hat Customer Portal].
+
+=== xPaaS Image for Red Hat JBoss A-MQ
+
+With this release, Red Hat JBoss A-MQ is available as a containerized
+xPaaS image that is designed for use with OpenShift Enterprise 3.0. It
+allows developers to quickly deploy an A-MQ message broker in a hybrid
+cloud environment.
+
+However, there are significant differences in supported configurations and functionality in the JBoss A-MQ image
+compared to the regular release of JBoss A-MQ. Documentation for other
+JBoss A-MQ functionality not specific to the JBoss A-MQ xPaaS image can
+be found in the
+https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_A-MQ/[JBoss
+A-MQ documentation on the Red Hat Customer Portal].
+
+== Known Issues for xPaaS Images
+
+These are the current known issues along with any known workarounds.
+
+*JWS*
+
+* Tomcat's access log valve logs to file in container instead of
+stdout.:
+https://issues.jboss.org/browse/CLOUD-57[https://issues.jboss.org/browse/CLOUD-57]‌
+
+Due to this issue, the logging data is not available for the central
+logging facility. To workaround this issues, use `oc exec` command to
+get the contents of the log file.
```

Comment:
Comment applies to any multi-paragraph/block bullets that follow as well (there appear to be many).